repos/fdbt-netex-output/src/netex-convertor/period-tickets/periodTicketNetexHelpers.ts

```diff
@@ -1052,7 +1052,11 @@ export const getPeriodConditionsElement = (
                             ? 'endOfFareDay'
```

Comment:
it looks like if `product.productValidity === 'endOfCalendarDay' || product.productValidity === 'fareDayEnd'`

then we are setting UsageEnd to endOfFareDay - which doesnt seem right

Reply:
@NitishAroraKPMG could you investigate which values are allowed in this field? I think for endOfCalendarDay we need to be setting it to something different.

I think try setting the value to endOfCalendarDay and see if the netex allows it. And if it doesnt, it should hopefully tell us which values are allowed.